lesson_2_loop_logic_operators_functions/while-for/whilefor_1.js

```diff
@@ -0,0 +1,8 @@
+// https://learn.javascript.ru/task/loop-last-value
+// Какое последнее значение выведет этот код? Почему?
+
+let i = 3
+
+while (i) {
+	alert(i--)
```

Comment:
Format this code. Code inside loop's body should be aligned with one tabulation.